setup

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# -e (-o errexit): exit immediately on error
+# -u (-o nounset): treat unset variables as erros
+# -o pipefail: propagate intermediate errors in pipelines
+set -euo pipefail
+
+VAULT_PASSWORD_FILE=~/.forem_selfhost_ansible_vault_password
+INVENTORY_FILE=inventory/forem/setup.yml
+
+main() {
+  print_step "Verifying that pip is available"
+  pip=$(command -v pip3 || echo -n "")
```

Comment:
I simplified this check to only check for `pip3` since that's already hardcoded in the Ansible playbooks anyway.